packages/coinstac-simulator/config/default.json

```diff
@@ -0,0 +1,13 @@
+{
+  "backend": false,
```

Comment:
FTR ☝️, @rssk _suggested_ the sub-option.  i drew the conclusion that `coinstac-simulator/config/default.json` should configure `coinstac-simulator`, and hence felt it was misleading if it was actually configuring PDBS